gas/testsuite/gas/riscv/cv-hwloop-23.s

```diff
@@ -0,0 +1,3 @@
+# Loop count must be positive integer in range:[0, 4094]
+target:
+	cv.setupi 0, 4095, 4
```

Comment:
Something's wrong with this test. The second operand of `cv.setupi` is `uimmS` (5-bit imm) and should be in the range `[0,30]`.

Did you mean to write
```suggestion
# Loop count must be positive integer in range:[0, 4094]
target:
	cv.setupi 0, 4, 4095
```

?

This still does not emit the above error message, though. The 3rd operand in `cv.setupi` has not to be aligned on a (half) word boundary.

If the error message, you expect in `gas/testsuite/gas/riscv/cv-hwloop-23.l` should be emitted, this test has to be
```suggestion
# Loop count must be positive integer in range:[0, 4094]
target:
	cv.setupi 0, 4, 4096
```

Reply:
@flip1995 The right mneumonic for cv.setupi is L, uimmL, uimmS. These are good questions to ask us as we will have come across this before. The documentation is often incorrect, when you come across these the best thing to do is check the the pulp binutils-gdb to see how it is implemented.

Reply:
Ah, that's good to know. Then I will fix this in LLVM.

Reply:
@flip1995 Following on from this, I think the tests are fine. Please let me know if you have any further questions :)

Reply:
Something with the test is still wrong though. I guess you meant to test `4096 ∉ [0, 4095]`?

Reply:
@flip1995 Ahh yes, good spotting, will change this!

Reply:
Just fixed the error - this was down to an incorrect comment. Thanks @flip1995